src/js/components/Widgets/Testimonial.jsx

```diff
@@ -29,31 +29,35 @@ class Testimonial extends React.Component {
 
   render () {
     renderLog(__filename);
-    const { testimonialAuthor, testimonial } = this.props;
+    const { testimonialAuthor, imageUrl, testimonial } = this.props;
 
     return (
-      <TestimonialContainer>
-        <ImageHandler
-          className="card-main__avatar__testimonial"
-          imageUrl={this.props.imageUrl}
-          alt="candidate-photo"
-          kind_of_ballot_item="CANDIDATE"
-        />
-        <TestimonialAuthor>
-          {testimonialAuthor}
-        </TestimonialAuthor>
-        <TextStyled>
-          <FormatQuote style={{
-            transform: 'rotate(180deg)',
-            verticalAlign: 'text-bottom',
-            position: 'relative',
-            top: '5px',
-            marginLeft: '-4px',
-          }}
-          />
-          {testimonial}
-        </TextStyled>
-      </TestimonialContainer>
+      <div className="card">
```

Comment:
Hi @jarodpeachey, we don't want to add <div className="card"><div className="card-main"> to every place that uses the Testimonial. Those divs can wrap <Testimonial /> on the Values page. Please search the code base to see everywhere <Testimonial is already included to understand the context. Thank you!